src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/packages.config

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AutoMapper" version="6.0.2" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.ApiManagement" version="3.4.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.ApiManagement" version="4.0.2-preview" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
```

Comment:
@solankisamir please remove `Microsoft.Azure.KeyVault.Core` and `Newtonsoft.Json` from this file